test/testsyncengine.cpp

```diff
@@ -119,6 +119,34 @@ private slots:
         QVERIFY(itemDidCompleteSuccessfully(completeSpy, "a3.eml"));
         QCOMPARE(fakeFolder.currentLocalState(), fakeFolder.currentRemoteState());
     }
+
+    void testRemoteChangeInMovedFolder() {
+        // issue #5192
+        FakeFolder fakeFolder{FileInfo{ QString(), {
+            FileInfo { QStringLiteral("folder"), {
+                FileInfo{ QStringLiteral("folderA"), { { QStringLiteral("file.txt"), 400 } } },
+                QStringLiteral("folderB")
+            }
+        }}}};
+
+        fakeFolder.syncOnce();
```

Comment:
There is already a syncOnce in the FakeFolder constructor, this shouldn't be necessary in every test.